unicorn_mode/samples/persistent/Makefile

```diff
@@ -48,3 +48,6 @@ debug: harness-debug.o
 
 fuzz: harness
 	../../../afl-fuzz -m none -i sample_inputs -o out -- ./harness @@
+
+debugmake:
```

Comment:
why not `debug` but `debugmake`?
Also what's the content of _LRT?

Reply:
who added this? @hexcoder- ? not me :)

Reply:
yes, i did. The make target 'debug' was used already in other Makefiles, so i used something unique for debugging the Makefile.
The content of _LRT is some unique string ('_QINU_'), following by the output of 'uname -s'.
The following is the only portable way (known to me) to do conditional logic (works for POSIX, gnu and bsdmake).
`$(_LRT:$(_UNIQ)Linux=-lrt)` means if _LRT ends with '_QINU_Linux', return '-lrt', otherwise return $_LRT.
The next expression $(__LRT:$(_UNIQ)$(UNAME_S)=) means, if __LRT ends in the same as _LRT, return '' otherwise return $__LRT.
The effect is, that $LRT is '-lrt' on Linux, and empty otherwise.

Reply:
makefile magician :)